pyanaconda/network.py

```diff
@@ -537,6 +538,18 @@ def add_connection_for_ksdata(networkdata, devname):
         values.append(['connection', 'id', devname, 's'])
         values.append(['connection', 'interface-name', devname, 's'])
 
+        # Add s390 settings
```

Comment:
I thought it is not necessary but now I think we don't want to call the stuff on non-s390 arch, and we are already checking for s390 arch in the module, so I'll add the check.